app/src/main/res/layout/activity_main.xml

```diff
@@ -108,9 +109,9 @@
         android:layout_gravity="center"
         android:background="@color/themeColor"
         android:gravity="center"
-        android:text="BMI Calculator"
+        android:text="@string/bmi_calculator"
         android:textColor="@color/white"
-        android:textSize="20dp"
+        android:textSize="20sp"
         android:textStyle="bold" />
```

Comment:
Fixed the error, redesigned the page layout and added the missing strings.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk